sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -574,6 +574,60 @@ __ESIMD_API
 #endif // !__ESIMD_FORCE_STATELESS_MEM
 }
 
+template <typename T, int NElts, cache_hint L1H = cache_hint::none,
+          cache_hint L2H = cache_hint::none, typename FlagsT>
+__ESIMD_API std::enable_if_t<__ESIMD_NS::is_simd_flag_type_v<FlagsT>>
```

Comment:
nit: here and in other places "__ESIMD_NS::" is not required. You can simply write "simd_mask<1> pred"

Reply:
fixed in latest commit, thx